### PR TITLE
blacklistd: Reduce diffs with upstream

### PR DESCRIPTION
**Pull request by jlduran**
@emaste: Here are a few commits that further reduce the diffs with upstream blacklistd.

Thank you!

### CODE REVIEW

File: contrib/blacklist/libexec/blacklistd-helper [hunk 1]
```diff
@@ -81,7 +81,7 @@ add)
 		    /sbin/pfctl -a "$2/$6" -f -
 		# insert $ip/$mask into per-protocol/port anchored table
 		/sbin/pfctl -qa "$2/$6" -t "port$6" -T add "$addr/$mask" && \
-		    /sbin/pfctl -q -k $addr && echo OK
+		    /sbin/pfctl -qk "$addr" && echo OK
```
Comment (emaste) [inline contrib/blacklist/libexec/blacklistd-helper:84]: All else being equal I like `-q -k` better, but even more than that I like avoiding diffs with upstream.

Comment (jlduran) [reply contrib/blacklist/libexec/blacklistd-helper:84]: 😂I had that exact same thought on the change below, but @zoulasc knows better.

Comment (zoulasc) [reply contrib/blacklist/libexec/blacklistd-helper:84]: he does not, he just followed suit with the rest of the code that combined options. he can change it.

Comment (jlduran) [reply contrib/blacklist/libexec/blacklistd-helper:84]: No worries. Thank you, we’ll abide!

**Comment by jlduran** [comment]
We are only a couple of commits behind upstream, the overall spirit of this pull request has been achieved.
Thank you both @emaste and @zoulasc for your cooperation!

**Pull request closed without merging**